guides/common/modules/proc_setting-up-flatpak.adoc

```diff
@@ -0,0 +1,12 @@
+[id="setting-up-flatpak_{context}"]
+= Setting up Flatpak
+
+This procedure installs the Flatpak package manager.
```

Comment:
I'd appreciate some information on where to install this. Some procedures are to be performed on the server, some on a proxy, some on a host. For this one, it's not entirely clear. So you could either specify it.. or you could probably actually include this as a prerequisite in "Enabling the Red Hat Flatpak remote".

Reply:
I've asked @sjha4 about this for clarification.

Reply:
This section is for the host.